lib/src/rules/missing_whitespace_between_adjacent_strings.dart

```diff
@@ -119,6 +94,61 @@ class _Visitor extends RecursiveAstVisitor<void> {
   }
 }
 
+extension on StringLiteral {
+  /// Returns whether this ends with whitespace, where any
+  /// [InterpolationExpression] counts as whitespace.
+  bool get endsWithWhitespace {
+    if (this is SimpleStringLiteral) {
```

Comment:
Consider assigning `this` to a local variable. Konstantin's been using `self` as the name of the local in such cases.

Reply:
Done